src/Logic.hs

```diff
@@ -296,15 +296,16 @@ clearPullRequest prId pr state =
 handleEventInternal
   :: TriggerConfiguration
   -> ProjectConfiguration
+  -> MergeWindowExemptionConfiguration
   -> Event
   -> ProjectState
   -> Action ProjectState
-handleEventInternal triggerConfig projectConfig event = case event of
+handleEventInternal triggerConfig projectConfig mergeWindowExemption event = case event of
   PullRequestOpened pr branch baseBranch sha title author -> handlePullRequestOpened pr branch baseBranch sha title author
   PullRequestCommitChanged pr sha -> handlePullRequestCommitChanged pr sha
   PullRequestClosed pr            -> handlePullRequestClosedByUser pr
   PullRequestEdited pr title baseBranch -> handlePullRequestEdited pr title baseBranch
-  CommentAdded pr author body     -> handleCommentAdded triggerConfig projectConfig pr author body
+  CommentAdded pr author body     -> handleCommentAdded triggerConfig projectConfig mergeWindowExemption pr author body
```

Comment:
Same nit about line length, although I see that the untouched case `PullRequestOpened` has it too. I think it's better to try to limit the line length to 100 characters if it's not too much hassle.